ansible/roles_ocp_workloads/ocp4_workload_lpe_3scale/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
If you don't have vars you can delete that whole directory.

Reply:
Resolved